javascript/javascript3/week1/weather-app.js

```diff
@@ -0,0 +1,65 @@
+const weatherInput = document.getElementById("weather-input");
+const weatherButton = document.getElementById("input-button");
+
+const loading = document.getElementById("loading");
+loading.style.visibility = "hidden"; //loading icon is hidden
```

Comment:
I really like how you did this. I would have done the same myself. Try to make a function (eg. to toggle loading screen) that does both of these CSS actions. Alternatively, attach a CSS class with hidden and 5% width to the element.